bindings/libdnf/rpm.i

```diff
@@ -19,6 +19,7 @@
     #include "libdnf/rpm/config_repo.hpp"
     #include "libdnf/rpm/nevra.hpp"
     #include "libdnf/rpm/package.hpp"
+    #include "libdnf/rpm/package_query.hpp"
```

Comment:
I suggest that in commit message it is supposed to write a note why the change was created.